apps/toolkit/src/components/ToolkitActiveViewersPanel.vue

```diff
@@ -18,14 +22,76 @@ onMounted(() => store.fetchActiveViewers());
     <v-divider class="mx-2" />
     <v-card-text class="pa-0">
       <v-list density="compact">
-        <v-list-item
-          v-for="viewer in store.activeViewers"
-          :key="viewer.id"
-        >
-          <v-list-item-title>{{ viewer.display }}</v-list-item-title>
-          <v-list-item-subtitle class="mt-n1"><small>{{ viewer.role }}</small></v-list-item-subtitle>
-        </v-list-item>
+        <v-dialog v-for="viewer in store.activeViewers" max-width="50rem" transition="dialog-bottom-transition">
+          <template v-slot:activator="{ props }">
+            <v-list-item :key="viewer.id" v-bind="props">
+              <v-list-item-title>{{ viewer.display }}</v-list-item-title>
+              <v-list-item-subtitle class="mt-n1"><small>{{ viewer.role }}</small></v-list-item-subtitle>
+
+              <template v-if="isNaN(viewer.id)" #append>
```

Comment:
Is there a better way to check for YouTube vs Twitch users without relying on `isNaN`?

Reply:
I don’t think so, I haven’t found a better way because Streamer.bot doesn’t give this information